test/index.test.js

```diff
@@ -9,18 +9,230 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-
 /* eslint-env mocha */
-
 const assert = require('assert');
-const { retrofit } = require('./utils.js');
+const { Request, Response } = require('@adobe/helix-universal');
+const proxyquire = require('proxyquire');
+
 const { main } = require('../src/index.js');
 
-const index = retrofit(main);
+const TEST_DATA = [];
```

Comment:
It would be nicer to have the test data as a separate file or something that is not generated.

Reply:
why?

Reply:
So that you can copy and paste them into other projects for testing and have a better chance at eyeballing the errors.